link.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from scrapy.linkextractors import LinkExtractor
+from scrapy.http import HtmlResponse, TextResponse, Response
+from six.moves.urllib.parse import urlparse
+import glob
+from timeit import default_timer as timer
+import click
+import io
+
+
+def main():
+    start = timer()
+
+    url = 'http://scrapinghub.com/'
+    link_extractor = LinkExtractor()
+    total = 0
+    for files in glob.glob('sites/index*'):
+
+        f = (io.open(files, "r", encoding="ISO-8859-1"))
+        html = "'''" + f.read() + "'''"
+
+        r3 = HtmlResponse(url=url, body=html, encoding='utf8')
+        links = link_extractor.extract_links(r3)
+        total = total + len(links)
+    end = timer()
+    print("\nTotal number of links extracted = {0}".format(total))
+    print("Time taken = {0}".format(end - start))
+    click.secho("Rate of link extraction : {0} links/second\n".format(
+        float(total / (end - start))), bold=True)
+
+    g = open("Stats.txt", 'a')
```

Comment:
See comment about ``AvSpeed.txt`` above - it would be nice to have a common way of passing benchmark results.